src/promptflow-azure/tests/sdk_cli_azure_test/e2etests/test_run_operations.py

```diff
@@ -1464,6 +1465,46 @@ def assert_func(details_dict):
 
         assert_batch_run_result(run, pf, assert_func)
 
+    @pytest.mark.skipif(not is_live(), reason="Content change in submission time which lead to recording issue.")
+    def test_model_config_obj_in_init(self, pf):
+        def assert_func(details_dict):
+            return details_dict["outputs.azure_open_ai_model_config_azure_endpoint"] == [
+                "https://gpt-test-eus.openai.azure.com/",
```

Comment:
Is this a good practice to have such values?

Reply:
Removed